kbchat/test_config.example.yaml

```diff
@@ -20,7 +20,10 @@ config:
   teams:
     acme:
       # A real team that you add your alice1, alice2, and bob1 all into
```

Comment:
i can't actually comment up on the line i care about, but what's up with the numbers on the usernames? it's weird. do we really need to test what happens on a user with multiple paperkeys? it doesn't look like we have any of those, and this would read much nicer with just `alice` and `bob` and `charlie`.